models/JobModel.js

```diff
@@ -1,9 +1,22 @@
 const mongoose = require("mongoose");
-
+const { constants } = require("../helpers/constants");
 const Schema = mongoose.Schema;
 const DataTypes = Schema.Types;
 const SkillSchema = require("./SkillModel");
 
+const applicantSchema = new Schema({
+  candidate: {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7787263)
```suggestion
	candidate: {
```